apps/openmw/mwgui/tradeitemmodel.cpp

```diff
@@ -24,6 +24,18 @@ namespace MWGui
         return mItems[index];
     }
 
+    ItemModel::ModelIndex TradeItemModel::getIndex(ItemStack item)
+    {
+        size_t i = 0;
+        for (std::vector<ItemStack>::iterator it = mItems.begin(); it != mItems.end(); ++it)
+        {
+            if (*it == item)
+                return i;
+            ++i;
+        }
+        return -1;
```

Comment:
```std::distance(std::begin(), std::find())``` would be more appropriate. It compiles to the same thing for ```std::vector```, but it also handles other container types that don't use contiguous memory.